tests/unit/test_wheel.py

```diff
@@ -243,15 +243,15 @@ def assert_permission(self, path, mode):
         target_mode = os.stat(path).st_mode & 0o777
         assert (target_mode & mode) == mode, oct(target_mode)
```

Comment:
Looking at this again, I'd test like this. @uranusjr what do you think?

```suggestion
        assert target_mode == mode, oct(target_mode)
```

Reply:
Hm, that's probably necessary for Windows. I guess we can improve this later.

Reply:
Okay. In that case, I will leave this unchanged.

Reply:
@sbidoul Yeah, on Windows `chmod()` is no-op so we need this additional mask to only check the positive bits, not negative.

Reply:
Given that no change is needed here, I am going ahead and resolving this conversation